packages/gateway/package.json

```diff
@@ -3,12 +3,10 @@
     "version": "1.0.0",
     "main": "index.js",
     "scripts": {
+        "rover": "rover supergraph compose --config ./supergraph-config.yaml > supergraph.graphql",
         "test": "echo \"Error: no test specified\" && exit 1"
     },
     "author": "",
     "license": "ISC",
-    "description": "",
-    "dependencies": {
-        "@apollo/gateway": "^0.41.0"
```

Comment:
I was wondering about whether these gateway/rover deps should be specific to this package - having not done much with workspaces yet, is there a best practice for where to include deps in a workspace set up?

Reply:
Yeah I think all shared at top level and all specific in nested packages, so i think i've done the wrong thing here, it should have stayed. I'll re-work it